.github/workflows/main.yml

```diff
@@ -21,3 +21,14 @@ jobs:
     - name: Run unit tests
       run: |
           yarn test:unit
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Deploy site to Fleek
+        id: deploy
+        uses: fleekhq/action-deploy@v1
+        with:
+          apiKey: ${{ secrets.FLEEK_API_KEY }}
```

Comment:
@midnight-madman another nice one! Thank you!

@appleseed-iii do we have the FLEEK_API_KEY yet from fleek?

Reply:
this is done